scripts/install.sh

```diff
@@ -172,8 +172,14 @@ then
 fi
 if [ "${platform}" = "Debian" ]
 then
+  if ! check adduser
+  then
+    echo "Could not find ${bold}adduser${normal} in \$PATH,"
+    echo "it is required for the Debian package install."
```

Comment:
```suggestion
    echo "Could not find ${bold}adduser${normal} in \$PATH."
```

Reply:
It's clear if we need if we err. Given that we don't say that for other errors, I'd just remove it here.

Reply:
Yeah I hesitated. It's not really obvious why `adduser` would be needed here (compared to sudo or curl). I thought adding some rational in the error would be at the same time a hint for the user and for the script maintainer.

Reply:
But you're not saying *why* it's needed. That would help. But it's obvious that it's needed if we bail out.

Reply:
haha, I actually don't know _why_, I only know _who_ 😄

Reply:
I can blame but not explain nor solve!

Reply:
User or developer first? Your call. 😉

Reply:
developer relationship as a comment, user relationship to stdout!